src/Microsoft.DotNet.Interactive.Formatting/PlainTextSummaryFormatter.cs

```diff
@@ -8,6 +8,7 @@ namespace Microsoft.DotNet.Interactive.Formatting;
 
 public static class PlainTextSummaryFormatter
 {
+    // FIX: (PlainTextSummaryFormatter) rename this
     public const string MimeType = "text/plain+summary";
```

Comment:
Again, I am guessing you intend to fix this up before merge. If not, I wonder if it may be better to log an issue to track this...

Reply:
That's a bigger change because it affects the VS Code extension. I've opened #3078 to track it.